### PR TITLE
[4.0] Change width in com_workflow

### PR DESCRIPTION
**Pull request by rjharishabh**
### Summary of Changes
Add `col-md-6` class

### Testing Instructions
`Dashboard` > `Content` > `Workflows` > `Click on any workflow`

### Actual result BEFORE applying this Pull Request
before-workflow

### Expected result AFTER applying this Pull Request
after-workflow

### Documentation Changes Required
None

### CODE REVIEW

File: administrator/components/com_workflow/tmpl/workflow/edit.php [hunk 1]
```diff
@@ -37,7 +37,7 @@
 	<?php // Add the translation of the workflow item title when client is administrator ?>
 	<?php if ($clientId === 0 && $this->item->id != 0) : ?>
 		<div class="row title-alias form-vertical mb-3">
-			<div class="col-12">
+			<div class="col-12 col-md-6">
```
Comment (Quy) [inline administrator/components/com_workflow/tmpl/workflow/edit.php]:
```suggestion
			<div class="col-md-6">
```
Everything below `md` will automatically be 12 thus no need to specify.

**Comment by brianteeman** [comment]
Did you work out why the field is even displayed?

**Comment by rjharishabh** [comment]
> Did you work out why the field is even displayed?

Yes, translation of the workflow item title

**Comment by brianteeman** [comment]
Really? Will it show you multiple languages?

**Comment by rjharishabh** [comment]
I think, When multiple languages are present, in the above image `en-GB`

**Comment by brianteeman** [comment]
I have tried and I can only ever get it to show one

**Comment by infograf768** [comment]
maybe: debug lang on, switch backend language to fr-FR for example. The pack is j3.

can’t test as not on desktop

**Comment by brianteeman** [comment]
Just one language

image

**Comment by infograf768** [comment]
That’s it.
If you create the missing strings in the admin language, then you should get the translation.
If I remember well, the name of. the workflow should also be a string constant.
Same behavior as custom admin menu items.

**Comment by brianteeman** [comment]
But shouldnt it display the value of the constant in all admin languages ?

**Comment by brianteeman** [comment]
Maybe I am confused and that was never a feature. I was just expecting to see a readonly field with the text for each language on the site

**Comment by ceford** [comment]
I have tested this item :white_check_mark: successfully on 145969b57a3f3c5719a6bd38705bd7ea94640ec5<br /><br />Sticking to the point - this pr does make the Name (en-GB) field the same width as the Name field. But ...<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33581">issues.joomla.org/tracker/joomla-cms/33581</a>.</sub>

**Comment by ceford** [comment]
The form clearly needs some work. It is not clear what the disabled field is for and I get this for the Basic Workflow:
screen shot 2021-05-09 at 07 22 53<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33581">issues.joomla.org/tracker/joomla-cms/33581</a>.</sub>

**Comment by ghost** [comment]
I have tested this item :white_check_mark: successfully on 145969b57a3f3c5719a6bd38705bd7ea94640ec5<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33581">issues.joomla.org/tracker/joomla-cms/33581</a>.</sub>

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33581">issues.joomla.org/tracker/joomla-cms/33581</a>.</sub>

**Comment by richard67** [comment]
Thanks!

**Comment by rjharishabh** [comment]
Thanks everyone

**Pull request merged**